tika-parsers/src/main/resources/org/apache/tika/parser/captioning/tf/Im2txtRestDockerfile

```diff
@@ -0,0 +1,62 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+FROM inception-rest-tika
+MAINTAINER Apache Tika Team
+
+#Install python pillow. TODO: Remove this with the fix for TIKA 2398
+RUN pip install pillow
+
+# Download the pretrained im2txt checkpoint
+WORKDIR /usr/share/apache-tika/models/dl/image/caption/
+
+RUN \
+    wget https://www.dropbox.com/s/l9ignjpjk774n2z/model_main_untuned.zip?dl=0 \
```

Comment:
can we please check in the model to https://github.com/USCDataScience/img2text.git? I can create the repo for you. Then please check in any scripts you used to generate the model. Then we can check in the model zip file.